libipl/libipl.H

```diff
@@ -82,6 +89,16 @@ void ipl_log(int loglevel, const char *fmt, ...);
 
 void ipl_set_error_callback_func(ipl_error_callback_func_t fn);
 
+/*
+ * @Brief Process PLAT error to add callout/deconfig and callback
+ * detailsipl_error_type
+ * @param fapiRC - FAPI return code
+ * @param target - target pointer
+ * @param deconfig - true deconfig the target 
+ */
+void ipl_process_fapi_error(const fapi2::ReturnCode& fapirc,
```

Comment:
is it an external or internal API? I mean the client is going to call this API?
Looks, you used it for internal purposes, right?

Reply:
agree it is internally called in the ipl code, but did not find a way to do this as ipl code does not have namespaces